src/ring_theory/free_ring.lean

```diff
@@ -41,7 +41,7 @@ section lift
 variables {β : Type v} [ring β] (f : α → β)
 
 def lift : free_ring α → β :=
-free_abelian_group.lift $ λ L, (L.map f).prod
+free_abelian_group.lift $ λ L, (list.map f L).prod
```

Comment:
Why this change?

Reply:
Because we have `free_monoid.map` now. I can use `free_monoid.lift α β f` but I wanted to minimize modifications outside of `free_monoid.lean`.